### DISCUSSION TITLE
У меня несколько вопросов, по кнопкам, по связям в таблице, по счетчику

### DISCUSSION THREAD

**Discussion by IrinaG1090** [totumonline/totum-ru-issues-and-discussions#437] [Как это сделать?] [answered]
Всем привет!!! Буду признательна, если подскажите или подсветите куда посмотреть в обучающем видео или в документации по следующим задачам. А то у меня совсем затык, перерыла документацию, пересмотрела много роликов на ютуб-канале Тотума.
Задачи:
1. Какой прописать код действия для кнопки, чтоб при нажатии на нее в расчетной таблице в цикле создавалась ячейка с текущим временем на момент нажатия? т.е. нажали на кнопку - текущее время зафиксировалось в созданную ячейку именно в расчетной таблице в цикле, а не в самом цикле создавалась строка.

2. Какую функцию можно использовать чтоб сделать счетчик подсчета количества строк в простой таблице, не задействуя период (дата от, дата до)

3. Можно ли сделать так чтоб строки с полем чекбокс улетали из одной таблицы и мигрировали в другую (например, есть цикл, в ней есть строки где чекбокс = true. (сама галочка в чекбоксах ставится в расчетной таблице в цикле, а в цикле просто отображен список, где чекбокс = true, чекбокс = false ) Нужно чтоб все эти строки из цикла улетали в простую таблицу. Может через кнопку это прописать, может еще как возможно. 

4. Как сделать производственный календарь на год с указанием часов работы сотрудников?

Подсветите пожалуйста куда посмотреть на какие функции или может есть не выложенное видео.

**Comment by ttm-alexeypolunin** [comment, marked as answer]
1. set в таблицу циклов where: id = $#nci
2. listCount, что бы взять list со всеми строками selectList без where - но если умвас там бесконечная таблица это умрет рано или поздно. Для таких случаев делается гдето-то поле в котором плюсуется или минусуется количество при доб/удалении
3. InsertList
4. https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/62

**Reply by IrinaG1090** [reply]
Благодарю, вас, Алексей. 
Буду пробовать. 
Результатами решения поделюсь в будущем
